doc/tutorials/workshop/content/exercises/12-common-rules.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: All types of rules
+PrevPage: 11-node-rules
+NextPage: ../finish
+---
+Common types of rules
+===================
+
+The rules we created in the previous chapter leveraged the `yamlfile_value`
+and the `file_permissions` template from CaC/content project. There are many
+more templates available in the project and each one enable us to quickly
+create rules with a specific behavior.
+
+In this section we will go through the most common checks when assessing the
+security posture of a cluster and the templates used to implement them.
+
+### Checking Kubernetes resources
+
+For example, to ensure that anonyous check for 
```

Comment:
Zombie sentence?

Reply:
This sections still needs to extended a bit more.